docs/guide/deployment/coap-gateway.md

```diff
@@ -137,6 +137,8 @@ CoAP API as specified in the [Open Connectivity Foundation - Device to Cloud Ser
 | `api.coap.externalAddress` | string | `External address including public domain/IP for coap client connection.` | `"coap-gw.example.com:5684"` |
 | `api.coap.maxMessageSize` | int | `Max message size which can be send/receive via coap. i.e. 256*1024 = 262144 bytes.` | `262144` |
 | `api.coap.ownerCacheExpiration` | string | `Time limit of how long to keep subscribed to device updates after last use of the given cache item.` | `1m` |
+| `api.grpc.subscriptionCacheExpiration` | string | `Time limit of how long to keep initialized events in subscription cache for duplicating events.` | `1m` |
```

Comment:
This depends - if lot's of events comes eg registering 10000 devices with 42 resources then 1 minute is too small because it's needed to be bigger because registration of so much devices take sometime (5+ minutes)

Reply:
It's not a good design if it's not deterministic on our end. We cannot modify deployment now and then just because more or fewer devices are being registered. What you confirmed is that it's kept there for the time when you subscribe and do retrieve to RD. Why execution of these 2 operations cannot define the time of the "cache"? Why you cannot just subscribe, process events, call rpc call, merge and return to client merged?

Reply:
As discussed, including current state will be deprecated and cache is no more required.